feat: slider

ddivyansh18 opened the pull request:
- Added slider
![image](https://user-images.githubusercontent.com/41231917/154835073-bb3ddab5-8b39-4904-aeee-5dff3e72f4dd.png)

ddivyansh18 committed:
feat: slider

- Added slider

ddivyansh18 committed:
feat: slider

thegeorgenikhil commented on Components/Slider/slider.css:
```diff
@@ -0,0 +1,34 @@
+  width: 100%;
+  height: 0.4rem;
+  border-radius: 0%;
+  background: lightgray;
+  outline: none;
```
Better to use hex code instead of colour names.

ddivyansh18 replied:
Thanks for the suggestions ! Just wanted to know for information, why hex codes are better than colour names ?

thegeorgenikhil replied:
Not sure but I think hex codes are universally supported in all browsers and these color names came up recently so some browser may have issues interpreting these name.

ddivyansh18 replied:
Slider background now uses a color variable

thegeorgenikhil commented on Components/Navigation/navigation.html:
```diff
@@ -88,10 +88,10 @@ <h1>App</h1>
-        <br />
-        <br />
         <p>
+          <br />
+          <br />
+          <br />
           Lorem Ipsum is simply dummy text of the printing and typesetting
```
Use padding insted of `br` tag as it is not semantic as well as bad for screen readers. [Link](https://stackoverflow.com/questions/1726073/is-it-sometimes-bad-to-use-br#:~:text=The%20main%20reason%20for%20not,space%20the%20blocks%20out%20properly.)

ddivyansh18 replied:
Agreed, will fix

ddivyansh18 replied:
@thegeorgenikhil  Removed all br tags from the code

thegeorgenikhil reviewed (approved):
Issues
- hex codes instead of color names
- use of br tags

Everything else looks good to merge! Do the required changes. Approving this.

Hnh013 reviewed (approved):
looks good to merge !

ddivyansh18 committed:
fix: removed br tags, unnecessary CSS imports

- Removed all the br tags from the code
- Removed unnecessary CSS imports

ddivyansh18 committed:
fix: br tags, slider color

- Removed br tags from docs
- Slider background color now uses a CSS variable for color

ddivyansh18 committed:
fix: slider color variable

- Slider background now uses a color variable

ddivyansh18 committed:
fix: slider code link

- Slider code link now shows correct code
